framework/include/lw_aifw/AIServiceCore.h

```diff
@@ -21,8 +21,8 @@
 #include <signal.h>
 #include "lw_aifw/lw_aifw.h"
 #include "lw_aifw/AIDataSource.h"
-#include "AIDataBuffer.h"
-#include "AIModel.h"
+#include "../../src/lw_aifw/include/AIDataBuffer.h"
+#include "../../src/lw_aifw/include/AIModel.h"
```

Comment:
Public header needs them, then are not they public headers?

Reply:
These are not public headers, they are used by AI Framework public headers but not by applications. Public header file declares their objects for its internal implementation.